master/internal/command/events.go

```diff
@@ -44,17 +48,63 @@ type event struct {
 	LogEvent *string `json:"log_event"`
 }
 
+type logSubscribers = map[*actor.Ref]webAPI.LogsRequest // CHECK could we get different pointers to
+// the same ref? thus resulting in different map keys
+
+// GetEventCount is an actor message used to get the number of events in buffer.
+type GetEventCount struct{}
+
 type eventManager struct {
-	bufferSize int
-	buffer     *ring.Ring
-	closed     bool
-	seq        int
+	bufferSize   int
+	buffer       *ring.Ring
+	closed       bool
+	seq          int
+	isTerminated bool // DISCUSS If we don't want to don't keep track of this here we'd either need to
+	// expose event externally or define and send a new message.
```

Comment:
DISCUSS If we don't want to don't keep track of this here we'd either need to
expose event externally or define and send a new message.